blazor/pdfviewer/getting-started/web-assembly-application.md

```diff
@@ -15,19 +15,27 @@ This section briefly explains about how to include [Blazor PDF Viewer](https://w
 
 * [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
 
-## Create a new Blazor App in Visual Studio
+## Integrate PDF Viewer into Blazor WebAssembly App
+
+1. Start Visual Studio and select **Create a new project**.
+2. For a Blazor WebAssembly experience, choose the **Blazor WebAssembly App** template and select **Next**. 
+![Create-a-new-WASM-app](GettingStarted_images/Create-new-WASM-app.png)
+3. Provide a **Project name** and confirm that the *Location* is correct. Select **Next**.
+![Set-project-name](GettingStarted_images/Set-WASM-project-name.png)
+4. In **Additional information dialog**, set target framework.  
+![Addition-information-WASM](GettingStarted_images/Additional_information_WASM.png)
 
 You can create **Blazor WebAssembly App** using Visual Studio in one of the following ways,
```

Comment:
![image](https://user-images.githubusercontent.com/20012243/183419908-6598d031-dd49-42ba-8200-f92c85be2d30.png)
These contents are not needed. Please remove it

Reply:
Removed the above contents